content/en/docs/armory-agent/agent-options.md

```diff
@@ -11,38 +11,58 @@ description: >
 
 ## Where to configure the Agent
 
-Set these options at the agent level in the `kubesvc.yaml` configuration file. If deploying as a non-Spinnaker<sup>TM</sup> service, you need to specify a `clouddriver.grpc` endpoint (e.g. `grpc.spinnaker.example.com:443`).
+On Kubernetes, configure the Agent using a `ConfigMap`. See the Quickstart's [Configure the Agent]({{< ref "armory-agent-quick#configure-the-agent" >}}) section for an example.
 
-## Kubernetes account
+If deploying as a non-Armory-Enterprise service, you need to specify a `clouddriver.grpc` endpoint (e.g. `grpc.spinnaker.example.com:443`) in your `kubesvc.yaml` file.
 
-At a minimum you need to add an account, give it a name, and set its Spinnaker permissions.
+## Configure Kubernetes accounts
+
+Kubernetes V2 provider accounts can be configured in multiple places:
+
+* Clouddriver files
+* Spring Cloud Config Server reading accounts from Git, Vault, or another supported backend
+* Plugins
+
+To avoid potential conflicts, Armory recommends migrating to the Agent all Kubernetes accounts other than those defined in a plugin.
```

Comment:
Accounts can coexist in multiple sources without problems, so this line can be deleted.